homeassistant/components/vera/__init__.py

```diff
@@ -72,10 +72,14 @@
     """Do setup of vera."""
     config = config_entry.data
 
-    # Get Vera specific configuration.
+    # Copy the configuration.yml options into the options.
     base_url = config.get(CONF_CONTROLLER)
-    light_ids = config_entry.options.get(CONF_LIGHTS, [])
-    exclude_ids = config_entry.options.get(CONF_EXCLUDE, [])
+    light_ids = config_entry.options.setdefault(
```

Comment:
We're not allowed to update the options directly. We need to use `hass.config_entries.async_update_entry` to update the options.